static/js/vue-cdr-access/src/components/full_record/abstract.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <p v-if="truncateAbstract" class="abstract">{{ truncatedAbstractText }}... <a class="abstract-text" @click.prevent="toggleAbstractDisplay()"
+                                                                                  href="#">{{ abstractLinkText }}</a></p>
+    <p v-else class="abstract">{{ recordData.briefObject.abstractText }}</p>
+</template>
+
+<script>
+import fullRecordUtils from "../../mixins/fullRecordUtils";
```

Comment:
I'm guessing this is just being included for `toggleAbstractDisplay`? If this component is the only place using that method now, I would suggest moving it into the `abstract.vue` component and dropping this import

Reply:
It's included for `toggleAbstractDisplay`, `truncateAbstract`, `truncateAbstractText`, and `AbstractLinkText`